sdl2promela/sdl2promela.py

```diff
@@ -18,29 +16,125 @@
 __log = logging.getLogger("sdl2promela")
 
 
-def __parse_arguments():
-    parser = argparse.ArgumentParser(description="SDL to Promela converter")
-    parser.add_argument(
-        "--sdl",
-        dest="files",
-        type=str,
-        nargs="+",
-        help="SDL system description; can contain only a single process",
-        default=[],
-    )
-    parser.add_argument(
-        "-o", "--output", dest="output_filename", type=str, help="output file name"
-    )
-    parser.add_argument("-v", "--verbose", action="store_true", help="verbose output")
-    parser.add_argument("--version", action="version", version=__version__)
-    parser.add_argument(
-        "--scl",
-        help="Read input files as Stop Conditions",
-        nargs="+",
-        type=str,
-        dest="scl_files",
-    )
-    return parser.parse_args()
+class ProgramOptions:
+    """Sdl2Promela Commandline Options"""
+
+    output_filname: str
+    """Output file."""
+    verbose: bool
+    """Show verbose output."""
+    sdl_files: List[List[str]]
+    """Groups of sdl files, every group shall contains one process."""
+    scl_files: [str]
+    """List of Stop Condition Language Files."""
+
+    def __init__(self):
+        self.output_filename = None
+        self.verbose = False
+        self.sdl_files = []
+        self.scl_files = []
+
+    def verify(self) -> bool:
```

Comment:
Doc

Reply:
Doc added, implementation changed (this was stub).